projects/gomods/athens/upload-athens-artifacts.sh

```diff
@@ -21,15 +21,17 @@ err_report() {
 }
 trap 'err_report $LINENO' ERR
 
+MAKE_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd -P)"
```

Comment:
This isn't the root of the Makefile right? I think doing a `pwd -P` might be enough here, and then changing the below lines to add an addition `../`